### DISCUSSION TITLE
Latest versions on REST API Reference missing endpoints?

### DISCUSSION THREAD

**Discussion by unicornsoftwareinc** [ant-media/Ant-Media-Server#6566] [Q&A]
Hi there, I was looking at the REST API Reference ( https://antmedia.io/rest/?urls.primaryName=2.10.0 ) and noticed that the latest few versions seem to be missing some endpoints. 

For example, it looks like the ManagementRestService section is missing from 2.10.0 and 2.9.0, but it is visible in versions 2.7.0 and below. I'm assuming these endpoints do still exist on the latest versions of Ant Media, but it's just the documentation is missing this?

**Comment by amarantmeida** [comment]
@unicornsoftwareinc Thank you for reporting this. We have created an [issue](https://github.com/ant-media/Ant-Media-Server/issues/6571) for the same and please follow it for more updates.

**Comment by muratugureminoglu** [comment]
Hi @unicornsoftwareinc 

Thank you for the issue you have reported, and I would like to inform you that this problem has been fixed.

Regards.
